src/Razor/test/Microsoft.VisualStudio.Editor.Razor.Test/DefaultVisualStudioRazorParserTest.cs

```diff
@@ -43,7 +43,7 @@ public DefaultVisualStudioRazorParserTest(ITestOutputHelper testOutput)
     private VisualStudioDocumentTracker CreateDocumentTracker(bool isSupportedProject = true, int versionNumber = 0)
     {
         var documentTracker = Mock.Of<VisualStudioDocumentTracker>(tracker =>
-        tracker.TextBuffer == new TestTextBuffer(new StringTextSnapshot(string.Empty, versionNumber)) &&
+        tracker.TextBuffer == new TestTextBuffer(new StringTextSnapshot(string.Empty, versionNumber), /* contentType */ null) &&
```

Comment:
```suggestion
        tracker.TextBuffer == new TestTextBuffer(new StringTextSnapshot(string.Empty, versionNumber), contentType: null) &&
```

Reply:
Can't use named parameters in expression trees.

Reply:
Sounds like an odd language limitation for me(

Reply:
Expression trees exactly represent syntax. This would mean that the named parameter needs to be reflected in the tree. I can't speak to why that wasn't added originally, but it certainly doesn't work today :slightly_smiling_face:.